rsconnect/bundle.py

```diff
@@ -656,7 +693,7 @@ def make_notebook_html_bundle(
         bundle_add_buffer(bundle, filename, output)
 
         # manifest
-        manifest = make_html_manifest(filename, image)
+        manifest = make_html_manifest(filename, image, env_management_py, env_management_r)
```

Comment:
It's a pity that there are _so_ many functions that this change needs to affect. Not for this PR, but I wonder if there's any way to reduce repetition here, or if it's already as clean as it can be. Just adds more surface area for errors, I feel like. ¯\\\_(ツ)\_/¯

Reply:
Completely agree. It would be nice if we relied more on **kwargs throughout. There are a couple places we do this already with **locals() but it’d be nice to be more consistent